machine-learning/pyproject.toml

```diff
@@ -22,6 +22,7 @@ fastapi = "^0.95.2"
 uvicorn = {extras = ["standard"], version = "^0.22.0"}
 pydantic = "^1.10.8"
 aiocache = "^0.12.1"
+python-multipart = "^0.0.6"
```

Comment:
seems like this is required if you are deal with form-data

Reply:
When you do the request, you can skip the form data part and just call `createReadStream(input.imagePath).pipe(res)`.

Reply:
I can get rid of it, but aren't we planning to send more information along with file, so we will be adding this back later ?

Reply:
You're right that it won't work with additional metadata. In this case it would be better to send it through gRPC instead, but that's something we can look into when we get there.

Reply:
I think using form data is fine, personally.